lib/src/base-styles/_nx-page-header.scss

```diff
@@ -39,61 +40,41 @@ $main-header-product-version-row-height: 1fr;
 .nx-product {
   align-self: center;
   box-sizing: border-box;
-  color: $nx-off-white;
-  display: -ms-grid;
-  display: grid;
-  grid-column-gap: $nx-spacing-xxs;
-  -ms-grid-rows: $main-header-product-title-row-height $main-header-product-version-row-height;
-  grid-template-rows: $main-header-product-title-row-height $main-header-product-version-row-height;
-  -ms-grid-columns: $main-header-product-height $nx-spacing-xxs auto;
-  grid-template-columns: $main-header-product-height auto;
-  grid-template-areas:
-    "logo title"
-    "logo version";
+  color: $main-header-color;
+  display: flex;
   height: $main-header-product-height;
   text-decoration: none;
   min-width: $nx-sidebar-width;
 }
 
-.nx-product__logo {
-  -ms-grid-row: 1;
-  -ms-grid-row-span: 2;
-  -ms-grid-column: 1;
-  -ms-grid-row-align: center;
+.nx-product__branding {
   align-self: center;
-  box-sizing: border-box;
-  grid-area: logo;
-  height: 100%;
-  width: 100%;
 }
 
-.nx-product__name {
-  -ms-grid-row: 1;
-  -ms-grid-column: 3;
-  @include semi-bold();
+.nx-product__wordmark {
+  height: 21px;
+  margin-top: 13px; // fix vertical centering
+}
 
-  -ms-grid-row-align: center;
+.nx-product__vertical-divider {
+  align-self: center;
+  color: #000;
+  font-size: 21px;
+  margin-left: $nx-spacing-sm;
+}
 
+.nx-product__name {
   align-self: center;
-  font-size: 23px;
-  grid-area: title;
-  -ms-grid-column-align: left;
-  justify-self: left;
-  line-height: normal;
-  text-transform: lowercase;
+  color: #474747;
+  font-size: 26px;
+  margin-left: $nx-spacing-sm;
   white-space: nowrap;
 }
 
 .nx-product__version {
-  -ms-grid-row: 2;
-  -ms-grid-column: 3;
-  @include bold();
-
-  -ms-grid-row-align: start;
-  align-self: start;
-  font-size: 11px;
-  grid-area: version;
-  -ms-grid-column-align: left;
-  justify-self: left;
-  line-height: 1.1;
+  align-self: center;
+  color: #545454;
+  font-size: 12px;
+  margin-top: 10px; // move version down to match product name baseline
```

Comment:
In zeplin their baselines aren't aligned

Reply:
> In zeplin their baselines aren't aligned

https://github.com/sonatype/sonatype-react-shared-components/pull/30/commits/ff2a27a1893f6060ef3056c61cb2527edd740ca5